README.md

```diff
@@ -41,6 +41,20 @@ If you happened to use `git clone` only, you won't see the `ORB_SLAM3` folder an
 
 `git submodule update --init`
 
+#### (optional) Using Canon Images
+
+Once downloaded, you can enter the repo and create a docker container for building either an `arm64` or `amd64` binaries  using Viam's canon tool by running one of the following commands:
```

Comment:
Would it make sense to expand on `Once downloaded` to something like what you said in the PR description?

```
First install it via:

go install github.com/viamrobotics/canon@latest
```